### DISCUSSION TITLE
Version 1.14.0 > weather display ?

### DISCUSSION THREAD

**Discussion by cr9christi9n** [InfiniTimeOrg/InfiniTime#2147] [General]
Hi there,
I just update my InfiniTime Watch from 1.11.0 to 1.14.0 and I ask my self where is the weather icone and view.
I found menu to choose "metric" or "imperial" but not the way to display weather !?
Many thanks in advance.

**Comment by FintasticMan** [comment]
You can display the weather in 1.14 in the PineTimeStyle watch face. If you tap and hold on the watch face, you can choose settings and then weather. You'll also need to make sure that you've enabled weather support in your companion app.
